src/ide/vscode/textLine.test.ts

```diff
@@ -13,7 +13,9 @@ const whiteSpaceTests: [string, number, number][] = [
   [" foo ", 1, 4],
 ];
 
-suite("TextLine", () => {
+suite("TextLine", function () {
+  this.timeout("100s");
+  this.retries(5);
```

Comment:
This is to fix CI for now.  I think the proper fix will be https://github.com/cursorless-dev/cursorless/issues/1158 so we don't keep forgetting to do this